### PR TITLE
Fix bugs

### PR DESCRIPTION
**Pull request by hshiah**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -651,15 +651,15 @@ Adds a reminder to the Financial Planner.
...
-Example of usage: `addreminder /t debt /d 2023.12.11`
+Example of usage: `addreminder /t debt /d 11/12/2023`
 
 Example output:
 ```
 You have added Reminder 
    Type: debt
-   Date: 2023.12.11
+   Date: 11/12/2023
    Status: Not Done
```
Comment (NeoMinWei) [inline docs/UserGuide.md:663]: There is still a Left Days portion below the status

**Review by ryan1604 — APPROVED** [review]
Thanks LGTM

**Review by NeoMinWei — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (ryan1604) [reply docs/UserGuide.md:663]: good find

**Pull request merged**